vehicle/ford/identity.go

```diff
@@ -31,14 +31,21 @@ var loginHeaders = map[string]string{
 	"User-Agent":      "Mozilla/5.0 (iPhone; CPU iPhone OS 17_0 like Mac OS X) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/17.1 Mobile/15E148 Safari/604.1",
 }
 
-var OAuth2Config = &oauth2.Config{
-	ClientID: ClientID,
-	Endpoint: oauth2.Endpoint{
-		AuthURL:  fmt.Sprintf("%s/oauth2/v2.0/authorize", LoginUri),
-		TokenURL: fmt.Sprintf("%s/oauth2/v2.0/token", LoginUri),
-	},
-	RedirectURL: "fordapp://userauthorized",
-	Scopes:      []string{ClientID, "openid"},
+func NewOauth2Config(domain string) *oauth2.Config {
```

Comment:
```suggestion
func OAuth2Config(domain string) *oauth2.Config {
```

try to don't add the prefix "new" to the Authentication Config, in the future it is not necessary to know about, that this is the new OAuth2Config method, because this is the current used method

Reply:
Not sure I follow. Oauth2Config typically is a struct. Here it‘s a func that provides a struct?

Reply:
Yes, this the same code :) only prevent the "new"-suffix :) (only has a suggestion)